lightningd/invoice.c

```diff
@@ -104,6 +104,29 @@ static bool hsm_sign_b11(const u5 *u5bytes,
 	return true;
 }
 
+/* We allow a string, or a literal number, for labels */
+static struct json_escaped *json_tok_label(const tal_t *ctx,
+					   const char *buffer,
+					   const jsmntok_t *tok)
+{
+	struct json_escaped *label;
+
+	label = json_tok_escaped_string(ctx, buffer, tok);
+	if (label)
+		return label;
+
+	/* Allow literal numbers */
+	if (tok->type != JSMN_PRIMITIVE)
+		return NULL;
+
+	for (int i = tok->start; i < tok->end; i++)
+		if (!cisdigit(buffer[i]))
+			return NULL;
+
+	return json_escaped_string_(ctx, buffer + tok->start,
+				    tok->end - tok->end);
```

Comment:
`tok->end - tok->end` == `0`?

Reply:
This seems to be the cause of the CI failure...

Reply:
Oops!